forge-gui/res/cardsfolder/upcoming/the_flood_of_mars.txt

```diff
@@ -0,0 +1,12 @@
+Name:The Flood of Mars
+ManaCost:2 U U
+Types:Creature Alien Zombie Horror
+PT:3/3
+K:Islandwalk
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ DBCounter | TriggerDescription$ Water Always Wins — Whenever CARDNAME attacks, put a flood counter on another target creature or land. If it's a creature, it becomes a copy of The Flood of Mars. If it's a land, it becomes an Island in addition to its other types.
+SVar:DBCounter:DB$ PutCounter | ValidTgts$ Creature.Other,Land.Other | AITgts$ Creature.OppCtrl+notnamedThe Flood of Mars,Land.OppCtrl+nonIsland | TgtPrompt$ Select target creature or land | CounterType$ FLOOD | SubAbility$ TrigCopy
+SVar:TrigCopy:DB$ Clone | ConditionDefined$ ParentTarget | ConditionPresent$ Creature | Defined$ Self | CloneZone$ Battlefield | CloneTarget$ ParentTarget | Execute$ TrigAnimate
+SVar:TrigAnimate:DB$ Animate | ConditionDefined$ ParentTarget | ConditionPresent$ Land | Type$ Island | Duration$ Permanent | IsCurse$ True

```

Comment:
Now the copying is working, but the TrigAnimate isn't working.

Reply:
well you changed it to Execute$ there which doesn't fit

Reply:
Not working with SubAbility$ either.

Reply:
you're also missing Defined$ to tell it which object to affect